src/game/interactables_2/egg_utopia/cannon.c

```diff
@@ -262,7 +262,11 @@ NONMATCH("asm/non_matching/game/interactables_2/egg_utopia/sub_807E66C.inc", boo
     s32 biggerX, biggerY, temp2, temp3;
     s32 r4;
     s16 playerX, playerY;
+#ifndef NON_MATCHING
```

Comment:
Nice. I'm surprised this was working?

Reply:
Me too!
Well, it did crash for me, so... that's how I found it! 😅